src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,270 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: sa
+      changes:
+        - createSequence:
```

Comment:
See my other comment for why increment-by-one sequences shouldn't be used. To create a UUID generator in liquibase, you can do something like:
```yml
databaseChangeLog:
  - property:
      name: uuid
      value: random_uuid()
      dbms: h2
```